cmd/root.go

```diff
@@ -37,9 +39,10 @@ var (
 		RunE:    run,
 		Version: "v0.3.1",
```

Comment:
This should probably be a .4 release right? It might break existing log scraping implementations since new fields are added and time is now different.

Reply:
sure...  I'd say it is a feature release